test/pki/system/ca.pem

```diff
@@ -1,3 +1,6 @@
+# SPDX-FileCopyrightText: Red Hat, Inc.
+# SPDX-License-Identifier: GPL-2.0-or-later
+
```

Comment:
These files are generated by test/genpki. Generating new files will drop the headers
and fail the build. This is good enough for now, but we really need to add the license
headers automatically to the generated files, or use *.license files.

Reply:
I'll use *.license files then. It is better, I think.